test/proposals/TestProposals2.sol

```diff
@@ -0,0 +1,180 @@
+pragma solidity 0.8.19;
+
+import {console} from "@forge-std/console.sol";
+import {Test} from "@forge-std/Test.sol";
+
+import {Addresses} from "@test/proposals/Addresses.sol";
+import {Proposal} from "@test/proposals/proposalTypes/Proposal.sol";
+import {CrossChainProposal} from "@test/proposals/proposalTypes/CrossChainProposal.sol";
+
+import {mip01b as mip} from "@test/proposals/mips/mip01b.sol";
```

Comment:
Can we be more consistent with naming these? MIP-B01 should be named mip-b01.sol or MIP-B01.sol